<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>811</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Noto Looped Lao Light&quot;;
letter-spacing:1px;</string>
    </property>
    <property name="text">
     <string>Nama</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_daftar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:black;
color:white;
border-radius :5px;</string>
    </property>
    <property name="text">
     <string>DAFTAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nama">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px solid black;
border-radius:5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Noto Looped Lao Light&quot;;
letter-spacing:1px;</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_email">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px solid black;
border-radius:5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Noto Looped Lao Light&quot;;
letter-spacing:1px;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_password">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px solid black;
border-radius:5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Noto Looped Lao Light&quot;;
letter-spacing:1px;</string>
    </property>
    <property name="text">
     <string>ReType Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_password2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px solid black;
border-radius:5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Noto Looped Lao Light&quot;;
letter-spacing:1px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Noto Looped Lao Light&quot;;
letter-spacing:1px;</string>
    </property>
    <property name="text">
     <string>ROLE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px solid black;
border-radius:5px;
</string>
    </property>
    <item>
     <property name="text">
      <string>OWNER</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ADMINISTRATOR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EMPLOYEE</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_warning">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color :red;
text-align:center;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Back">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>490</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../icon/previous.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lTitle_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 10pt &quot;Noto Looped Lao Medium&quot;;

letter-spacing :2.5px;</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QLabel" name="lTitle_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>530</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 10pt &quot;Noto Looped Lao Medium&quot;;

letter-spacing :2px;</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Exit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>490</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../Resource Grapich/Resources-Icon/illustration-business-concept/logout.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#5941A9;</string>
    </property>
    <widget class="QLabel" name="lSubTitle_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 57 10pt &quot;Noto Looped Lao Medium&quot;;
letter-spacing:3px;</string>
     </property>
     <property name="text">
      <string>Silahkan isi data pengguna baru</string>
     </property>
    </widget>
    <widget class="QLabel" name="lTitle_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 57 22pt &quot;Noto Looped Lao Medium&quot;;
letter-spacing:2px;</string>
     </property>
     <property name="text">
      <string>Tambah Pengguna Baru</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>811</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#5941A9;</string>
     </property>
     <widget class="QLabel" name="lSubTitle_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font: 57 10pt &quot;Noto Looped Lao Medium&quot;;
letter-spacing:3px;</string>
      </property>
      <property name="text">
       <string>Silhkan pilih 2 menu dibawah ini</string>
      </property>
     </widget>
     <widget class="QLabel" name="lTitle_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font: 57 22pt &quot;Noto Looped Lao Medium&quot;;
letter-spacing:2px;</string>
      </property>
      <property name="text">
       <string>Pengaturan</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
